charts/zenko/values.yaml

```diff
@@ -88,3 +88,10 @@ redis:
     enabled: false
   metrics:
     enabled: true
+
+redis-ha:
+  rbac:
+    create: true
+  replicas:
+    servers: 5
```

Comment:
We switched to 3 replicas for our default deployments.

Reply:
Fixed.